python/lang/best-practice/unspecified-open-encoding.py

```diff
@@ -0,0 +1,77 @@
+def func1():
+    # ruleid:unspecified-open-encoding
+    fd = open('foo')
+    fd.close()
+
+def func2():
+    # ruleid:unspecified-open-encoding
+    fd = open('foo', mode="w")
+    fd.close()
+
+def func3():
+    import os
+    db_root="test"
+    # ruleid:unspecified-open-encoding
+    with open(os.path.join(db_root, "data.json")) as f:
+        i = 2
+
+def func4():
+    import os
+    # ruleid:unspecified-open-encoding
+    with open(os.path.join("test", "b", mode="b")) as f:
```

Comment:
Are the parentheses correct here?

Reply:
Or is this intended to test `mode` but NOT inside `open`?

Reply:
Yep, this was intended as a test to make sure only args to `open` are checked. A previous version of the rule looked for `mode` with a regex instead of the `pattern:` format so I added the test. Not necessary anymore - should I remove?

Reply:
Any test is a good test :D I was just asking to make sure I understood. 👍